### PR TITLE
Update RecoveryServices schema from swagger

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

**Comment by TianoMS** [comment]
@nilaydshah Could you take a look at this PR for RecoveryServices?

File: schemas/2016-06-01/Microsoft.RecoveryServices.json [hunk 1]
```diff
@@ -85,40 +82,25 @@
             {
               "type": "string",
               "enum": [
+                "Standard",
```
Comment (nilaydshah) [inline schemas/2016-06-01/Microsoft.RecoveryServices.json]: We don't support name as Standard now. Earlier BackupVault used to support Standard as name, which shouldn't be used now.

File: schemas/2016-06-01/Microsoft.RecoveryServices.json [hunk 2]
```diff
@@ -85,40 +82,25 @@
...
                 "RS0"
               ]
             },
             {
               "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
             }
           ],
-          "description": "Gets or sets the sku name. Required for vault creation, optional for update. Possible values include: 'RS0'"
-        },
-        "tier": {
```
Comment (nilaydshah) [inline schemas/2016-06-01/Microsoft.RecoveryServices.json]: tier is required parameter in sku and supported tier is Standard.

File: schemas/2016-06-01/Microsoft.RecoveryServices.json [hunk 3]
```diff
@@ -85,40 +82,25 @@
...
-            {
-              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-            }
-          ],
-          "description": "Gets or sets the sku tier. Required for vault creation, optional for update. Possible values include: 'Standard'"
+          "description": "The Sku name."
         }
       },
       "required": [
-        "name",
```
Comment (nilaydshah) [inline schemas/2016-06-01/Microsoft.RecoveryServices.json]: name and tier are required params (part of sku) for vault creation

File: schemas/2016-06-01/Microsoft.RecoveryServices.json [hunk 4]
```diff
@@ -85,40 +82,25 @@
...
-        "tier"
+        "name"
       ],
-      "description": "The SKU of the recovery services vault."
+      "description": "Identifies the unique system identifier for each Azure resource."
     },
-    "RecoveryServicesPropertiesCreateParameters": {
+    "VaultProperties": {
       "type": "object",
-      "properties": {
```
Comment (nilaydshah) [inline schemas/2016-06-01/Microsoft.RecoveryServices.json]: Vault creation requires empty properties bag. will removing this work without any issue?

**Comment by nilaydshah** [comment]
@praveennet can you also please take a look?

**Comment by TianoMS** [comment]
@nilaydshah i think the changes were generated from https://github.com/Azure/azure-rest-api-specs/blob/master/arm-recoveryservices/2016-06-01/swagger/recoveryservices.json. some of the issues may need to be fixed in swagger spec.

**Comment by nilaydshah** [comment]
@dragonfly91 can you take a look at comments in this PR?

**Comment by daschult** [comment]
@nilaydshah @TianoMS Tian is correct. These changes were generated from the [current swagger document](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json) for RecoveryServices. Please update the swagger document for any changes, and let me know when your changes have been merged into the Azure/azure-rest-api-specs repository.

**Comment by TianoMS** [comment]
@nilaydshah @dragonfly91 can you take a look at this PR?

**Comment by TianoMS** [comment]
@nilaydshah @dragonfly91 can you take a look at this PR?

**Pull request closed without merging**